core-js/src/main/javascript/cdf-legacy/components/filter/js/TreeFilter/views/scrollbar/MCustomScrollBarEngineImplementation.js

```diff
@@ -0,0 +1,40 @@
+/*!
```

Comment:
MCustomScrollBarEngineImplementation is a bit too long and repetitive. What about renaming the file (and the global variable) to
MCustomScrollBarEngine.js  ?